packages/css-data/src/tailwind-parser/substitute.test.ts

```diff
@@ -0,0 +1,129 @@
+import { describe, expect, it, jest } from "@jest/globals";
+
+import { substituteVariables } from "./substitute"; // Import the function
+
+const cssPreflight = `
+*,
+::before,
+::after {
+ --var-a: rgba(111, 111, 111, 0.5);
+ --var-b: rgba(222, 222, 222, 0.5);
+}
+`;
+
+describe("expandTailwindShorthand", () => {
+  it("resolves simple variable", () => {
```

Comment:
we are using test() everywhere

Reply:
`test` doesn't sound with almost any text

Reply:
its just not bdd, but its possible to write test() in a way that is well-worded